src/pages/Autolending/AutolendingWhen.vue

```diff
@@ -1,102 +1,100 @@
+				@lightbox-closed="showLightbox = false"
+			>
+				<div class="when-inputs-wrapper">
+					<lend-timing-dropdown />
+					<!--
 							While lenders with an auto-deposit donation should not have an auto-lending donation after
 							the fixes for AUTO-44 and AUTO-206, the check below includes donation===0
 							for graceful degradation, so that users with both donations still see the actual value.
 						-->
-						<p v-if="hasAutoDepositDonation && donation === 0">
-							Your auto-deposit includes a donation, so auto-lending donations are disabled.
-						</p>
-						<div v-else>
-							<kv-radio
-								data-test="is-autolending-donation-on"
-								id="is-autolending-donation-on"
-								radio-value="on"
-								v-model="donationToggle"
-							>
-								Include a donation to Kiva of
-								<kv-dropdown-rounded v-model="donation">
-									<option :value="0">
-										0%
-									</option>
-									<option :value="5">
-										5%
-									</option>
-									<option :value="10">
-										10%
-									</option>
-									<option :value="15">
-										15%
-									</option>
-									<option :value="20">
-										20%
-									</option>
-								</kv-dropdown-rounded>
-							</kv-radio>
-							<kv-radio
-								data-test="is-autolending-donation-off"
-								id="is-autolending-donation-off"
-								radio-value="off"
-								v-model="donationToggle"
-							>
-								No donation to Kiva.
-							</kv-radio>
-						</div>
-					</div>
-					<template slot="controls">
-						<kv-button
-							data-test="when-save-button"
-							class="smaller button"
-							v-if="!isSaving"
-							@click.native="save"
-							:disabled="!isChanged"
+					<p v-if="hasAutoDepositDonation && donation === 0">
+						Your auto-deposit includes a donation, so auto-lending donations are disabled.
+					</p>
+					<div v-else>
+						<kv-radio
+							data-test="is-autolending-donation-on"
+							id="is-autolending-donation-on"
+							radio-value="on"
+							v-model="donationToggle"
 						>
-							Save
-						</kv-button>
-						<kv-button data-test="when-save-button" class="smaller button" v-else>
-							Saving <kv-loading-spinner />
-						</kv-button>
-					</template>
-				</kv-lightbox>
+							Include a donation to Kiva of
+							<kv-dropdown-rounded v-model="donation">
+								<option :value="0">
+									0%
+								</option>
+								<option :value="5">
+									5%
+								</option>
+								<option :value="10">
+									10%
+								</option>
+								<option :value="15">
+									15%
+								</option>
+								<option :value="20">
+									20%
+								</option>
+							</kv-dropdown-rounded>
+						</kv-radio>
+						<kv-radio
+							data-test="is-autolending-donation-off"
+							id="is-autolending-donation-off"
+							radio-value="off"
+							v-model="donationToggle"
+						>
+							No donation to Kiva.
+						</kv-radio>
+					</div>
+				</div>
+				<template slot="controls">
+					<kv-button
```

Comment:
I may be missing some context, but instead of an @click on this button, a containing <form> element with an @submit handler would give more a11y context and allow a user to press enter to submit this form.

Reply:
@ryan-ludwig these are slots in a kvLightbox. To make the default slot of the lightbox and the controls slot both be contained in a form element would require some reworking of the lightbox component.

Reply:
@eddieferrer Ahh, that makes sense, thanks.